doc/grafana/README.md

```diff
@@ -0,0 +1,78 @@
+# Grafana dashboards for the DevWorkspace Operator
+
+This directory contains a sample Grafana dashboard that processes some of the prometheus metrics exposed by the DevWorkspace Operator. Once Prometheus and Grafana are configured with the DevWorkspace Operator's metrics server as a source, the dashboard can be imported from [json](grafana-dashbaord.json).
+
+## Testing DevWorkspace Operator metrics locally
+
+To quickly test metrics locally (without installing Prometheus Grafana in the cluster), it's possible to deploy prometheus and grafana as containers locally.
+
+1. Create a clusterrolebinding for a service account to allow the metrics-reader role (this example uses the controller's service account)
+    ```bash
+    NAMESPACE=devworkspace-controller # Namespace where controller is installed
+    kubectl create clusterrolebinding dw-metrics \
+      --clusterrole=devworkspace-controller-metrics-reader \
+      --serviceaccount=${NAMESPACE}:devworkspace-controller-serviceaccount
+    ```
+2. Get the serviceaccount's token to enable local access to metrics
+    ```bash
+    TOKEN=$(kubectl get secrets -o=json -n ${NAMESPACE} | jq -r '[.items[] |
+      select (
+        .type == "kubernetes.io/service-account-token" and
+        .metadata.annotations."kubernetes.io/service-account.name" == "devworkspace-controller-serviceaccount")][0].data.token' \
+      | base64 --decode)
+    ```
+3. In another terminal, use `kubectl port-forward` to expose the controller's service locally
+    ```bash
+    kubectl port-forward service/devworkspace-controller-manager-service 8443:8443 &;
```

Comment:
```
error: Service devworkspace-controller-manager-service does not have a service port 8443
[1]  + 30425 exit 1     kubectl port-forward service/devworkspace-controller-manager-service 8443:8443
```

```suggestion
    kubectl port-forward service/devworkspace-controller-metrics 8443:8443 &;
```

Reply:
Right, forgot we had to change that -- thanks!